helena.gemspec

```diff
@@ -19,20 +19,20 @@ survey/test development, longitudinal studies and instant feedback.'
   s.files = `git ls-files`.split("\n")
   s.test_files = `git ls-files -- {spec}/*`.split("\n")
 
-  s.add_dependency 'mongoid', '>= 4.0'
-  s.add_dependency 'mongoid-simple-tags', '>= 0.1.3'
-  s.add_dependency 'mongoid-tree', '>= 2.0'
-  s.add_dependency 'mongoid_orderable', '>= 5.2'
-  s.add_dependency 'rails', '>= 4.2'
-  s.add_dependency 'rails-i18n', '>= 4.0'
-  s.add_dependency 'responders', '>= 2.0'
-  s.add_dependency 'sass-rails', '>= 5.0'
-  s.add_dependency 'simple_form', '>= 4'
-  s.add_dependency 'slim', '>= 3.0'
+  s.add_dependency 'mongoid'
+  s.add_dependency 'mongoid-simple-tags'
+  s.add_dependency 'mongoid-tree'
+  s.add_dependency 'mongoid_orderable'
+  s.add_dependency 'rails', '~> 6.0.3.4'
+  s.add_dependency 'rails-i18n'
+  s.add_dependency 'responders'
+  s.add_dependency 'sass-rails'
+  s.add_dependency 'simple_form'
+  s.add_dependency 'slim'
```

Comment:
I'm not sure if dropping the constraints on the production dependencies is a good idea, but I also don't know which versions Helena really supports.

Reply:
Well, they all have `>=`, so I think they were put there to guard against older versions